lib/model/upi_account.dart

```diff
@@ -1,89 +1,87 @@
 import 'bank_account.dart';
 
 class UpiAccount {
```

Comment:
@rahuljain-rzp hope this is not part of session token changes

Reply:
No this is not @ramprasadAnand 
But non transaction flows were breaking because of missing bankId field in UpiAccount.  Now the android flutter app is working fine.